code/modules/vehicles_legacy/vehicle.dm

```diff
@@ -18,11 +18,12 @@
 	buckle_flags = BUCKLING_PASS_PROJECTILES_UPWARDS
 	buckle_lying = 0
 
+	var/health
+	var/maxhealth
```

Comment:
did we finish these conversions?

Reply:
nope

vehicle_legacy basically is pending "toss the fuck out of the codebase", that's why it's still there